Website/src/rendering/src/stories/Checkout/CheckoutCommon.ts

```diff
@@ -1,4 +1,5 @@
 import { Spec } from 'ordercloud-javascript-sdk';
+import { DeliveryTypes } from 'src/models/ordercloud/DOrder';
```

Comment:
Seems like build is failing due to a storybook error 

![image](https://user-images.githubusercontent.com/96472119/181033607-a5ad6b37-91f6-4e07-9b23-c0f5e9dd40c8.png)